include/boost/url/const_string.hpp

```diff
@@ -42,18 +44,17 @@ namespace urls {
     valid for the lifetime of the original
     object.
 */
-class const_string : public string_view
+class const_string
+    : public string_view
 {
-    struct base;
+    using ref_counter_t = std::atomic<std::size_t>;
 
-    base* p_ = nullptr;
+    ref_counter_t* refs_ = nullptr;
+    any_allocator<ref_counter_t> alloc_;
 
-    template<class Allocator>
-    base*
-    construct(
-        std::size_t n,
-        Allocator const& a,
-        char*& dest);
+    inline
```

Comment:
I thought we were gonna put `inline` on the definitions?

Reply:
I was. Then you forbid me and asked me to open an issue (https://github.com/CPPAlliance/url/issues/109).

Reply:
no I said don't change everything that's already there just open an issue for it (which you did), but we should use it for any new code or code that is modified as a result of other changes

Reply:
My understanding was this shouldn't be done unless we do it for boost.url, boost.json, and boost.beast for some reason. Anyway, I'm not going to look for the message. I'm just going to put `inline` in the implementation from now on then.